File: src/main/java/com/github/novusvetus/intellijstatsplugin/PluginMenu.java
package com.github.novusvetus.intellijstatsplugin;

import com.intellij.openapi.actionSystem.AnAction;
import com.intellij.openapi.actionSystem.AnActionEvent;
import com.intellij.openapi.project.Project;

public class PluginMenu extends AnAction {
    public PluginMenu() {
        super("IntelliJStatsPlugin Settings");
    }

    public void actionPerformed(AnActionEvent e) {
        Project project = e.getProject();
        Settings popup = new Settings(project);
        popup.show();
    }
}

File: src/main/kotlin/com/github/novusvetus/intellijstatsplugin/PluginMenu.kt
package com.github.novusvetus.intellijstatsplugin

import com.intellij.openapi.actionSystem.AnAction
import com.intellij.openapi.actionSystem.AnActionEvent

class PluginMenu : AnAction("IntelliJStatsPlugin Settings") {
    override fun actionPerformed(e: AnActionEvent) {
        val project = e.project
        val popup = Settings(project)
        popup.show()
    }
}